image.c

```diff
@@ -133,6 +143,141 @@ openImgdisplay()
     return FALSE;
 }
 
+static int
+protocol_test_environment(void)
+{
+    const char * envptr;
+
+    if ((envptr = getenv("TERM"))) {
+	/* The purpose of strncmp(3) where used below is like the glob `prefix*',
+	 * or the regex `^prefix' */
+	if (strcmp(envptr, "xterm-kitty") == 0)
+	    return INLINE_IMG_KITTY;
+	if (strcmp(envptr, "xterm-ghostty") == 0)
+	    return INLINE_IMG_KITTY;
+	if (strncmp(envptr, "mlterm", 6) == 0)
+	    return INLINE_IMG_OSC5379;
```

Comment:
I'm not in a position to test, but I was under the impression that MLterm had dropped support for OSC 5379 images several years ago. See the [3.8.5 release notes](https://github.com/arakiken/mlterm/blob/e6ac4a3d56fda42969425dcad6d179ae7f6ebeb6/doc/en/ReleaseNote#L301).

I may be misinterpreting what they're saying there, but I think it would be good idea to confirm this actually works (if you haven't already done so) before forcing the use of OSC 5379. MLterm does support Sixel, so you should still have that as a fallback.

Reply:
Huh, indeed. It's not fully gone, but you have to pass a flag when compiling:
./configure CFLAGS=-DENABLE_OSC5379PICTURE

I suppose it's better to stick with Sixel.